### DISCUSSION TITLE
"Ready for review" button on draft pull request is misleading

### DISCUSSION THREAD

**Discussion by jwindmuller** [community/community#66206] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Draft pull requests do not block reviewers from reviewing the code so naming the button "Ready for review" is a bit misleading.

If the design of the feature is to prevent people from reviewing the code while on draft then that functionality should be disabled, but being able to review the PR while it's a draft is actually really useful. So it seems like the button's label could be tweaked to match how the PR tool works. "Ready for merge" would achieve that.

As an extra improvement, clicking the "Ready for merge" could check if there have been changes since a reviewer approved the PR and re-request their review automatically.

**Comment by timginter** [comment]
or just "Ready", then it fits whatever the context people are using
